parlai/agents/transformer/transformer.py

```diff
@@ -3,7 +3,6 @@
 # LICENSE file in the root directory of this source tree.
 """Transformer Agents."""
 from parlai.core.agents import Agent
-from parlai.utils.misc import warn_once
```

Comment:
is this no longer needed in this file?

Reply:
Yeah in a previous commit in this PR, ```lint_changed.sh``` threw an error that it wasn't being used, so I got rid of it!